pkg/core/info/combat.go

```diff
@@ -42,6 +42,7 @@ type AttackInfo struct {
 	StrikeType       attacks.StrikeType
 	UseDef           bool    // we use this instead of flatdmg to make sure stat snapshotting works properly
 	UseHP            bool    // we use this instead of flatdmg to make sure stat snapshotting works properly
+	UseEM            bool    // we use this instead of flatdmg to make sure stat snapshotting works properly
```

Comment:
I think you should update the comment to signify that Lunar damages need to differentiate between EM MVs and flat damage